config/routes.rb

```diff
@@ -84,6 +84,8 @@
           patch :add_students
           patch :remove_organization
         end
+
+        resource :lti_configuration, controller: "orgs/lti_configurations"
```

Comment:
Looks like you aren't exposing all of the resource actions on `lti_configuration`, so maybe you should only allow `show`, `new`, and `create`?
```suggestion
        resource :lti_configuration, only: %i[show new create], controller: "orgs/lti_configurations"
```

Reply:
Can that be in the next PR when we work on those actions?